code/modules/clothing/under/jobs/medsci.dm

```diff
@@ -64,8 +64,8 @@
 	)
 
 /obj/item/clothing/under/rank/virologist
-	desc = "It's made of a special fiber that gives special protection against biohazards. It has a virologist rank stripe on it."
-	name = "moebius virologist's jumpsuit"
+	desc = "An old green shirt and khakis, originally worn by the Moebius Virology Corps, now it is discarded in maintenance after their disbandment."
+	name = "moebius virologist's uniform"
```

Comment:
```suggestion
	name = "moebius virologist's uniform"
	desc = "Old green shirt and khakis, originally worn by the Moebius Virology Division, now it is discarded in maintenance after their disbandment."
```
Imo shouldn't be in the game at all.
Far as I know Moe got 'divisions' instead of 'corps'.
Also `name` variable usually comes before `desc`. That will have no ill effects on the game, but having variables in order they have been defined is good in terms of code style.

Reply:
Tbh, virology should have been a part of Medical Division, rater than something independent.

Reply:
The idea was that it was a DIVISION of the Medical DEPARTMENT
I suppose, but I didn't really think about the logistics of that, it's just defunct now, so here's the uniform.
I plan to board up viro and turn it into a fun set piece in the future, cause right now it's NOT for some reason.